### PR TITLE
example: update 2-storage example

### PR DESCRIPTION
**Pull request by AnnaShaleva**
- Added .yaml configuration
- Updated imports
- Added en&ru workshop part

### CODE REVIEW

**Comment by AnnaShaleva** [comment]
Do we need workshop part for it?

**Comment by roman-khimov** [comment]
Yep. And we also have `CityOfZion` being mentioned in `1-print.go` and some README text.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ but it’s not the only thing we’re working on.
 ## NEO GO
 As you know network is composed of nodes. These nodes as of now have several implementations:
 - https://github.com/neo-project/neo
-- https://github.com/CityOfZion/neo-python
+- https://github.com/nspcc-dev/neo-python
```
Comment (roman-khimov) [inline README.md]: Nope.

File: README.ru.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 ## NEO GO
 Как вы знаете, сеть состоит из нод. В текущий момент ноды имеют несколько реализаций:
 - https://github.com/neo-project/neo.git
-- https://github.com/CityOfZion/neo-python
+- https://github.com/nspcc-dev/neo-python
```
Comment (roman-khimov) [inline README.ru.md]: Nope.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**